<?xml version="1.0"?>
<launch>
  <param name="/scale_linear" value="300"/>
  <param name="/scale_angular" value="200"/>

  <node pkg="usb_cam" type="usb_cam_node" name="usb_cam_node" output="screen"/>

  <node pkg="opencv_apps" type="hough_circles" name="hough_circles" output="screen">
    <remap from="/image" to="/usb_cam_node/image_raw"/>
    <param name="debug_view" value="false"/>
    <param name="accumulator_threshold" value="85"/>
  </node>
  <node pkg="image_view" type="image_view" name="image_view_output" output="screen">
    <remap from="/image" to="/hough_circles/debug_image"/>
  </node>

  <node pkg="stl_ros_sample" type="turtle_operation" name="turtle_operation" output="screen"/>

  <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node" output="screen"/>
</launch>
